Documentation/ColumnsConfig/Type/Input/Properties/Eval.rst

```diff
@@ -190,7 +190,7 @@ saving the record:
       'example_field' => [
          'config' => [
             'type' => 'text',
-            'eval' => 'trim,Vendor\\Extension\\Evaluation\\ExampleEvaluation,required'
+            'eval' => 'trim,required,' . \Vendor\Extension\Evaluation\ExampleEvaluation::class
```

Comment:
required has been moved to a separate property for V12, so we would have to take your changes as here to version 11 then move required to its own property for version 12

Reply:
Ok, what should I do? Maybe it's easier if I create separate PR for each branch or do you want to do this?

Reply:
I think in this case it is better to make one PR for each version. also we cannot backport to 10.4 as the structure changed. Could you do that?

Reply:
ok